bin/create_hugo_pages.py

```diff
@@ -30,9 +30,9 @@
 
 from docopt import docopt
 from glob import glob
-from tqdm import tqdm
 import logging as log
 import os
+from rich.progress import track
```

Comment:
Just curious about the tqdm → rich.progress transition, in case you care to comment.

Reply:
Rich is prettier and has more features. For example, the logging mechanism now also uses rich for pretty-printing, but at the same time this avoids log messages "interrupting" the progress bar (the log output appears above the progress bar instead). Also, the progress bars play nicely with CI, whereas the tqdm progress bars often caused screens full of output if you scrolled through the logs of a failed `check_build` run.

And finally, even pip has switched to rich. ;)